python/model/protobuf_flow/Data.aac

```diff
@@ -14,11 +14,11 @@ schema:
       type: Authentication
```

Comment:
Let's avoid injecting spaces into reference models -- this should be covered by the use of tests.

Reply:
That has been reverted in the latest push

Reply:
The diff is still showing this file as altered.